templates/hp_comware_display_device_manuinfo.textfsm

```diff
@@ -0,0 +1,21 @@
+Value Filldown CHASSIS_ID (\d+)
+Value Required SLOT_TYPE (Slot|Subslot|Fan|Power|Chassis)
+Value SLOT_ID (\d+|self)
+Value DEVICE_NAME (\S+)
+Value DEVICE_SERIAL_NUMBER (\S+)
+Value MANUFACTURING_DATE (\S+)
+Value VENDOR_NAME (\S+)
+Value MAC_ADDRESS (\S+)
+
+
+Start
+  ^\s?Chassis\s+${CHASSIS_ID}
+  ^\s?${SLOT_TYPE}\s+${SLOT_ID}
+  ^\s?DEVICE_ID
+  ^\s?The\s+operation\s+is\s+not\s+supported\s+ -> Record Start
+  ^\s?DEVICE_NAME\s*:\s*${DEVICE_NAME}
+  ^\s?DEVICE_SERIAL_NUMBER\s*:\s*${DEVICE_SERIAL_NUMBER}
+  ^\s?MAC_ADDRESS\s*:\s*${MAC_ADDRESS}
+  ^\s?MANUFACTURING_DATE\s*:\s*${MANUFACTURING_DATE}
+  ^\s?VENDOR_NAME\s*:\s*${VENDOR_NAME} -> Record Start
+  ^. -> Error
```

Comment:
Do you mind adding `  ^\s*$$` above the error line?

Do you mind also changing the beginning `\s?` to `\s*` just in case they do change to more than one space at the beginning?

Those should be the last changes and then we can get these merged in.

Reply:
I will follow the format next time.
Practice makes perfect. ^_^